docker/conf-workers/synapse.supervisord.conf.j2

```diff
@@ -0,0 +1,31 @@
+
+[program:synapse_main]
+command=/usr/local/bin/prefix-log /usr/local/bin/python -m synapse.app.homeserver {#
+#}  --config-path="{{ main_config_path }}" {#
+#}  --config-path=/conf/workers/shared.yaml
```

Comment:
this `{#\n#}` business is  is a bit grim. I believe supervisord will ignore linebreaks provided the next line is indented, so I think you can get rid of them.

Reply:
Well I feel a bit silly now — thanks. I overcomplicated it and that didn't work, so wound up with this (which I also don't like).